packages/aws-cdk/test/integ/cli.exclusions.js

```diff
@@ -14,7 +14,23 @@ For example:
 },
 
 */
-const exclusions = []
+const exclusions = [
+  {
+    "test": "test-cdk-deploy-with-parameters.sh",
+    "version": "1.31.0",
+    "justification": "This test doesn't use a unique sns topic name and it collides with our regular integ suite"
+  },
+  {
+    "test": "test-cdk-deploy-wildcard-with-parameters.sh",
+    "version": "1.31.0",
+    "justification": "This test doesn't use a unique sns topic name and it collides with our regular integ suite"
+  },
+  {
+    "test": "test-cdk-deploy-nested-stack-with-parameters.sh",
```

Comment:
https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/test/integ/cli/test-cdk-deploy-nested-stack-with-parameters.sh#L9
could you use STACK_NAME_PREFIX?

Reply:
Its running the tests from the previous tag so we can't change them.